infra/license/check-result-formatter.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {readFileSync, writeFileSync} from 'fs';
+import {EOL} from 'os';
+
+/**
+ * Argument List
+ *  - Input paths for each of license verification results
+ */
+var args = process.argv.slice(2)
+const resultPaths = args;
+
+var commentBody = [] as string[];
+var totalWarnCount = 0;
+var totalDenialCount = 0;
+for (let i = 0; i < resultPaths.length; ++i) {
```

Comment:
Q. why does this `i` is declared with `let`? 

other variables are `var` or `const`. AFAIK, ES6 recommends using `let` and `const`. 

Can I know your  principal line to distinuish `var` and `let`?

Reply:
I used them with following principles.

- `const` : variable whose value should/will not be changed
- `let` : temporary variables which is used in for-loop, while-loop, etc.
- `var` : Other than above

But as you said, it would be better to use `let` instead of `var`!
I will change `var` to `let`, thanks for knowledge :)
And I will create another PR to fix those things in another files :D

Reply:
I was just curious. You don't have to fix them. :-) 

We need to have a style guide for ts.

Reply:
I applied your suggestion because another trivial mistakes are found 😅

Reply:
I found this.
https://google.github.io/styleguide/jsguide.html#features-use-const-and-let